### ISSUE TITLE
[v1.14.0-rc.1] No /etc/nvidia-container-runtime/config.toml

### ISSUE THREAD

**Issue by nie3e** [NVIDIA/nvidia-container-toolkit#78]
ubuntu 22.04
After docker and nvidia drivers + cuda installation I am running:
```sh
distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
      && curl -s -L https://nvidia.github.io/libnvidia-container/experimental/$distribution/libnvidia-container.list | \
         sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
         sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
		 
sudo apt-get update
sudo apt-get install -y nvidia-container-toolkit
sudo nvidia-ctk runtime configure --runtime=docker
sudo systemctl restart docker
```

It installs `v1.14.0-rc.1` and there is no config file `/etc/nvidia-container-runtime/config.toml` - becase of this i can't use `swarm-resource`.
`v1.13.1` works fine.

**Comment by elezar** [comment]
@nie3e yes, this is a known issue. We provide a `nvidia-ctk config default` command to generate a default config -- instead of managing a per-distribution config in the packages themselves. This opens up the option to use the same packages across more distributions.

We have a fix in progress to generate the config as a post-install scriplet which should ensure that the file actually exists and this will be included in the `rc.2` that we are looking to release later this week.

Note that the config file is still read and the format hasn't changed, so you should be able to manually set up the config as before. Runnning `mkdir -p /etc/nvidia-container-runtime && nvidia-ctk config default > /etc/nvidia-container-runtime/config.toml` should create the config file with default values for you to edits.

**Comment by nie3e** [comment]
@elezar okay thanks for answer, gonna use this command.
